txscript/scriptbuilder_test.go

```diff
@@ -228,21 +228,22 @@ func TestScriptBuilderAddData(t *testing.T) {
 		// BIP0062: OP_PUSHDATA4 can never be used, as pushes over 520
 		// bytes are not allowed, and those below can be done using
 		// other operators.
-		{
-			name:     "push data len 521",
-			data:     bytes.Repeat([]byte{0x49}, 521),
-			expected: nil,
-		},
-		{
-			name:     "push data len 32767 (canonical)",
-			data:     bytes.Repeat([]byte{0x49}, 32767),
-			expected: nil,
-		},
-		{
-			name:     "push data len 65536 (canonical)",
-			data:     bytes.Repeat([]byte{0x49}, 65536),
-			expected: nil,
-		},
+		// EDIT: As per CHIP VM Limits, the 520 limit has been raised to 10000. Commenting the following lines for now.
```

Comment:
If we don't want these tests we should just delete them. Or have them scoped to before the upgrade so they run with legacy logic.